app/controllers/ChoiceController.scala

```diff
@@ -80,6 +80,8 @@ class ChoiceController @Inject()(
             proceedJourney(DisassociateUcrAnswers(), controllers.consolidations.routes.DisassociateDucrController.display())
           case forms.Choice.ShutMUCR =>
             proceedJourney(ShutMucrAnswers(None), routes.ChoiceController.displayPage())
+          case forms.Choice.ViewSubmissions =>
+            proceedJourney(ViewSubmissionsAnswers(), routes.ChoiceController.displayPage())
```

Comment:
Similar, we dont need Answers for this journey, just redirect to the relevant page